src/components/RadioOptions.svelte

```diff
@@ -13,6 +13,9 @@ const onInput = event => {
     <label>
       <input type="radio" {name} value={option.value} on:input={onInput} />
       {option.label}
+      {#if option.description}
+        <small>{option.description}</small>
```

Comment:
You can use our `Description` component here:
```suggestion
        <Description>{option.description}</Description>
```
That will get you consistent styling with our other such text.